packages/plugin-core/src/survey.ts

```diff
@@ -172,6 +214,158 @@ export class PriorToolsSurvey extends DendronSurvey {
   }
 }
 
+export class LapsedUserReasonSurvey extends DendronQuickPickSurvey {
+  async onAnswer(result: vscode.QuickPickItem) {
+    const label = result.label
+    let extra: string | undefined;
+    let reason: string | undefined;
+    switch(label) {
+      case "I haven't had time to start, but still want to.": {
+        reason = "time";
+        break;
+      }
+      case "I am not sure how to get started.": {
+        reason = "stuck";
+        break;
+      }
+      case "I've encountered a bug which stopped me from using Dendron.": {
+        // could you describe what happened?
+        reason = "bug";
+        extra = await vscode.window.showInputBox({
+          ignoreFocusOut: true,
+          placeHolder: "Type here",
+          prompt: "Could you describe, in simple words, what happened?",
+          title: label
+        });
+        break;
+      }
+      case "I found a different tool that suits me better.": {
+        reason = "tool";
+        extra = await vscode.window.showInputBox({
+          ignoreFocusOut: true,
+          placeHolder: "Type here",
+          prompt: "What feature was missing in Dendron for your use case?",
+          title: label
+        });
+        break;
+      }
+      case "Other": { // "Other"
+        reason = "other"
+        extra = await vscode.window.showInputBox({
+          ignoreFocusOut: true,
+          placeHolder: "Type here",
+          prompt: "Please freely type your reasons here.",
+          title: label,
+        });
+        break;
+      }
+      default: {
+        break;
+      }
+    }
+    
+    AnalyticsUtils.track(SurveyEvents.LapsedUserReasonAnswered, {
+      reason,
+      extra
+    });
+  }
+
+  onReject() {
+    AnalyticsUtils.track(SurveyEvents.LapsedUserReasonRejected);
+  }
+
+  static create() {
+    const title = "What is the reason you haven't started using Dendron yet?";
+    const choices = [
+      { label: "I haven't had time to start, but I still want to." },
+      { label: "I am not sure how to get started." },
+      { label: "I've encountered a bug which stopped me from using Dendron." },
+      { label: "I found a different tool that suits me better." },
+      { label: "Other" },
+    ];
+    return new LapsedUserReasonSurvey({ title, choices, canPickMany: false });
+  } 
+}
+
+export class LapsedUserOnboardingSurvey extends DendronQuickPickSurvey {
+  CALENDLY_URL = "https://calendly.com/d/mqtk-rf7q/onboard";
```

Comment:
we talked about giving folks the option to go to the getting started guide as an alternative to the one on one survey. is it worhtwile adding a multi-select so users can select one or both methods here?

Reply:
Forgot to point out some decisions I made here:

So this survey shows up when the lapsed user message is displayed, and then rejected. I think it would be a bit weird if we ask them here again when they already rejected the initial prompt like a minute ago.